signer/src/testing/dummy.rs

```diff
@@ -510,6 +512,30 @@ impl fake::Dummy<fake::Faker> for SigHash {
     }
 }
 
+impl fake::Dummy<std::ops::Range<u64>> for BitcoinBlockHeight {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(_config: &std::ops::Range<u64>, rng: &mut R) -> Self {
+        rng.next_u32().into()
+    }
+}
+
+impl fake::Dummy<std::ops::Range<u64>> for StacksBlockHeight {
+    fn dummy_with_rng<R: rand::Rng + ?Sized>(_config: &std::ops::Range<u64>, rng: &mut R) -> Self {
+        rng.next_u32().into()
+    }
+}
```

Comment:
Shouldn't those use `_config`? If not, do we need those two (vs the general version below)

Reply:
If tests were using a specific range previously, not using `_config` may be changing the test in some unintended way (eg, potentially generating only "smaller" numbers). Can you check if we are using the ranged version for anything meaningful, and if not switch to the other one?

Reply:
[b20069a](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/b20069a0b59d32acd4abdc5339cc8ec244aee4d5) and [62dac02](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/62dac029ff467df29f47f427e0124eb76e90521f)